### ISSUE TITLE
It's untested

### ISSUE THREAD

**Issue by dude8604** [dude8604/ESP3212-Eagle#1]
If you've tested it and/or see any mistakes, tell me here.

**Comment by hallard** [comment]
Hi @dude8604 
Thanks for this lib. I wanted to try it but It seems the pinout is different, the one you provided is the following pinout
![image](https://cloud.githubusercontent.com/assets/2471931/19717670/0be5824c-9b62-11e6-9a7c-45e9711f5e34.png)

this one http://www.electrodragon.com/w/ESP32 is the following 
![image](https://cloud.githubusercontent.com/assets/2471931/19717713/3fa65958-9b62-11e6-8c99-44450d2f4833.png)
so both are the same but in your library the footprint is

![image](https://cloud.githubusercontent.com/assets/2471931/19717747/6a501e3c-9b62-11e6-9b10-25016ab0003f.png)
It's not the same pinout and number of pins ?

Did I miss something ?

**Comment by dude8604** [comment]
Yes,the one you linked to is the ESP-WROOM-32 and it is a different footprint.  They just announced that the ESP3212 isn't actually going to be made.  So I guess that makes this library obsolete.

**Comment by hallard** [comment]
Whoo what a fast response ;-)

Humm that's interesting, I just ordered some ESP32 WROOM-32 from Olimex but also some [ESP3212](http://www.electrodragon.com/product/esp3212-esp-32s-iot-wifi-wifibluetooth-dual-core/) from ElectroDragon (which has the pinout I mention) so I'm curious to see what I will receive...
see full doc and pinout here http://www.electrodragon.com/w/ESP32
